src/main/java/org/EmailCarbonFootprint/EmailCarbonFootprintAssignment.java

```diff
@@ -1,35 +1,35 @@
 package org.EmailCarbonFootprint;
 
-import java.util.Map;
 import java.util.Scanner;
 
 import javax.mail.MessagingException;
 import javax.mail.Store;
 
 public class EmailCarbonFootprintAssignment {
-	
+
 	public static void main(String[] args) {
 		GmailAccountDetails gmailAccountDetails;
 		EmailCarbonFootprintCalculator emailCarbonFootprintCalculator;
-		Scanner scanner = new Scanner(System.in);
-        System.out.println("Enter the emailId:");
-        String emailId = scanner.next();
-        System.out.print("Enter your email password: ");
-        String password = scanner.next();
-        double inboxEmailsCount;
-        double sentEmailsCount;
-        double spamEmailsCount;
-        gmailAccountDetails = new GmailAccountDetails(emailId,password);
-        try {
-        	Store store = GmailConnection.getGmailConnection(gmailAccountDetails);
-        	inboxEmailsCount = gmailAccountDetails.getNumberOfInboxEmails(store);
-        	sentEmailsCount = gmailAccountDetails.getNumberOfSentEmails(store);
-        	spamEmailsCount = gmailAccountDetails.getNumberOfSpamEmails(store);
-        	emailCarbonFootprintCalculator = new EmailCarbonFootprintCalculator(emailId,inboxEmailsCount,sentEmailsCount,spamEmailsCount);
-        	emailCarbonFootprintCalculator.printCarbonFootprint();
-        }
-        catch(MessagingException exception) {
-              System.out.println(exception);	
-        }
-    }
+		Scanner input = new Scanner(System.in);
+		System.out.println("Enter the emailId:");
+		String emailId = input.next();
+		System.out.print("Enter your email password: ");
+		String password = input.next();
+		int inboxEmailsCount;
```

Comment:
you can use inboxEmailCount

Reference : 
inboxEmailsCount: This name suggests that the variable represents the count (number) of multiple inbox emails. It emphasizes that you are counting more than one email.

inboxEmailCount: This name suggests that the variable represents the count (number) of inbox emails. It implies that you are counting a single value, which is the total count of inbox emails.